cmd/process-agent/README.md

```diff
@@ -0,0 +1,44 @@
+# Datadog Process Agent
+
+## Installation
+
+See the [Live Processes docs](https://docs.datadoghq.com/graphing/infrastructure/process/#installation) for installation instructions.
+
+## Development or running from source
+
+Pre-requisites:
+
+* `go >= 1.10.1`
```

Comment:
Will this whole change set make us use a different version of go now?

Reply:
Good catch I will update this 👍